recipes/insilicoseq/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.3.5" %}
+{% set sha256 = "a90227ce29d6060acdf17ff6a3960add7566af824652cbbc26b8d0334e618df2" %}
+
+package:
+  name: insilicoseq
+  version: '{{version}}'
+
+source:
+  url: https://github.com/HadrienG/InSilicoSeq/archive/{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  noarch: python
+  entry_points:
+    - iss = iss.app:main
+  script: python -m pip install .
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - iss
+  requires:
+    - nose
```

Comment:
You can probably get rid of `nose`, if it were actually required to import `iss` it should be a dependency.

Reply:
Done.